### PR TITLE
Document Grafana plugin

### PR DESCRIPTION
**Pull request by iain-b**
https://deploy-preview-609--roadie.netlify.app/

### CODE REVIEW

**Comment by iain-b** [comment]
ops, I've omitted a section

**Review by punkle — DISMISSED** [review]

**Comment by iain-b** [comment]
Pushed a new commit need review again

File: content/docs/integrations/grafana/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Prerequisites
+
+You'll need a Grafana account with an API key and the url of your Grafana UI and API (if different).
+
+## Adding the plugin
+
+### Configure the Grafana endpoints
+
+Configure the grafana endpoints to use via `Administration -> Settings -> Grafana`. If you're using grafana.net your
```
Comment (Xantier) [inline content/docs/integrations/grafana/index.md:21]:
```suggestion
Configure the Grafana endpoints to use via `Administration -> Settings -> Grafana`. If you're using grafana.net your
```

File: content/docs/integrations/grafana/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+frontend and backend endpoints should be the same e.g. `https://<your-company>.grafana.net/`. If you're using hosting
+grafana yourself you'll need to specify a url to the frontend which is used by backstage to generate links and an API
```
Comment (Xantier) [inline content/docs/integrations/grafana/index.md:23]:
```suggestion
Grafana yourself you'll need to specify a URL to the frontend which is used by backstage to generate links and an API
```

File: content/docs/integrations/grafana/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Add the plugin to the UI
+
+The Grafana plugin provides two components which can be added to the Backstage UI. You must be a Backstage admin to
+add components to the UI.
+
+The `EntityGrafanaDashboardsCard` and `EntityGrafanaAlertsCard` components can be [added to catalog dashboards](/docs/getting-started/updating-the-ui/#updating-dashboards). These
+list dashboards and alerts respectively.
+
+### Set the grafana annotation on entities
```
Comment (Xantier) [inline content/docs/integrations/grafana/index.md:43]:
```suggestion
### Set the Grafana annotation on entities
```

File: content/docs/integrations/grafana/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+The grafana plugin uses an annotation to link entities in the Backstage catalog to data in Grafana. This annotation should
```
Comment (Xantier) [inline content/docs/integrations/grafana/index.md:45]:
```suggestion
The Grafana plugin uses an annotation to link entities in the Backstage catalog to data in Grafana. This annotation should
```

File: content/docs/integrations/grafana/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+have a tag in Grafana as its value. Any alerts or dashboards with this tag will be displayed once the annotation is set and 
+the components added to the UI.
+
+Make a PR to the following to your catalog-info.yaml file:
```
Comment (Xantier) [inline content/docs/integrations/grafana/index.md:49]:
```suggestion
Make a PR like the following to your catalog-info.yaml file:
```

**Pull request merged**